app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -72,27 +76,27 @@ fun ReviewScreen(
   }
   var textValue by remember { mutableStateOf(if (ownerHasReviewed) matchingReview?.text!! else "") }
 
-  val context = LocalContext.current // Get the current context
```

Comment:
You removed the documentation here.

Reply:
I don't think that documentation was necessary in the first place...

Reply:
Well you are right but it was just to mention it because removing documentation is kind of weird now that it is there. However we can argue that is not necessary so you are right.